src/main/java/Christina/Homework_1/WaitAndPrint.java

```diff
@@ -0,0 +1,24 @@
+package Christina.Homework_1;
+
+import java.util.Scanner;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by Christina Mnoyan on Sep, 2019
+ */
+
+public class WaitAndPrint {
+
+    public static void main(String[] args) throws InterruptedException {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter the time you want to wait:");
+        int timeToWait = sc.nextInt();
+        timeWaiter(timeToWait);
+    }
+
+    public static void timeWaiter(int timeToWait) throws InterruptedException {
+        TimeUnit.SECONDS.sleep(timeToWait);
```

Comment:
es spasum ei vor es xndirn aneq while-ov kam for-ov, ))

TimeUnit.SECONDS.sleep(timeToWait);

sa der chenq ancel ) bayc de chishta ashxatum, ban chem kara asem)